trento/adoc/checks_customization.adoc

```diff
@@ -0,0 +1,101 @@
+
+==== The check customization modal includes the following elements:
+
+* Selected Check ID.
+
+* Check description.
+
+* Warning message that Trento & SUSE are not liable for system malfunctions due to custom check values.
+
+* A list of all customizable check values. Each value includes a value name with the original default check value and an input field with the current customized or default value.
+
+* The current target specific provider.
+
+* Save, Reset and Close buttons at the bottom.
+
+The Save button is disabled by default and only becomes enabled when the user checks the warning and modifies a value. Pressing the Save button triggers a toast notification confirming successful check customization and the custom values are stored *persistently* in Wanda’s database. So even after a Reboot of the system the customized values are not lost.
+
+The Reset button remains disabled if the check is not customized.
+If the check has been customized previously, the Reset button inside the modal is enabled, allowing users to revert changes within the modal. When resetting the check, a confirmation dialog appears to confirm the action in order to avoid accidental resets. On a successful reset a toast notification appears that the check customization was reset.
+
+The modal closes automatically after saving or resetting the check.
+
+A Modified pill indicator appears next to the check ID, highlighting that the values have been customized. A Reset icon appears next to the settings icon, on the right side, allowing users to revert to default values after closing the modal. Pressing on the reset icon opens a confirmation dialog to confirm the action in order to avoid accidental resets.
+
+=== Target-Specific Check Customization
+
+A check is always executed on a target, which can be a host or a cluster. Users can customize check values specific to the target environment to ensure optimal system performance. Customizations are target-specific and do not affect other targets or the default check values in the YAML file.
+
+=== Required Permissions
+
+Only *admin users* or users with the `all:checks_customization` permission can customize checks. Users without this permission will not see the customization button in the Trento Web console. If a check was modified, the modified Pill is shown to all users.
+
+=== Customizable Checks
+
+By default, a check with expected values is customizable. This applies to all checks unless explicitly disabled in the check YAML file. It cannot be changed in the Trento Web console.
+
+Supported value types include:
+
+- Strings
+
+- Numbers
+
+- Booleans
+
+User input is validated to ensure that the input value matches the expected type before allowing to save the custom values in Wanda's database. If the input type is incorrect or mixed, the customization fails, triggering a toast notification, that checks customization failed and a warning message in the modal itself.
+
+=== Disable Check Customization
```

Comment:
This part is very optional, and less user relevant. Feel free to skip it, as discussed